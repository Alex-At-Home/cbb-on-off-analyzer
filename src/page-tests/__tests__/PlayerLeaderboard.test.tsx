import React from 'react';
import PlayerLeaderboardPage from '../../pages/PlayerLeaderboard';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe("PlayerLeaderboardPage", () => {
  test("PlayerLeaderboardPage - should create snapshot", () => {
    //TODO: this isn't the best test because it will just step over the async fetch
    //and render a trivial page, still will do for now

    const wrapper = shallow(<PlayerLeaderboardPage testMode={true}/>);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
