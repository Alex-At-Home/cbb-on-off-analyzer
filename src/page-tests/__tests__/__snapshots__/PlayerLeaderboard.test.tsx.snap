// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlayerLeaderboardPage PlayerLeaderboardPage - should create snapshot 1`] = `
<Container
  className="medium_screen"
  fluid={false}
>
  <Head>
    <meta
      content="https://server/thumbnails/player_leaderboard_thumbnail.png"
      property="og:image"
    />
    <meta
      content="https://server/thumbnails/player_leaderboard_thumbnail.png"
      name="twitter:image"
    />
  </Head>
  <Row
    noGutters={false}
  >
    <Col
      className="text-center"
      xs={12}
    >
      <h3>
        CBB Player Leaderboard
         
        <span
          className="badge badge-pill badge-info"
        >
          BETA!
        </span>
      </h3>
    </Col>
  </Row>
  <Row
    className="border-bottom"
    noGutters={false}
  >
    <HeaderBar
      common={
        Object {
          "gender": "",
          "year": "",
        }
      }
      thisPage="player-_leaderboard"
    />
  </Row>
  <Row
    className="mt-3"
    noGutters={false}
  >
    <PlayerLeaderboardTable
      dataEvent={
        Object {
          "confs": Array [],
          "lastUpdated": 0,
          "players": Array [],
          "teams": Object {},
        }
      }
      onChangeState={[Function]}
      startingState={Object {}}
    />
  </Row>
  <Footer
    dateOverride={0}
    server="server"
  />
</Container>
`;
